config.yaml

```diff
@@ -13,8 +13,174 @@ environment:
     auto_detect: True
 weather:
     station: mongo
+    aat_metdata:
+      name: 'AAT metdata'
+      link: 'ftp://site-ftp.aao.gov.au/pub/local/metdata/metdata1.dat'
+      column_names:  ['time_UTC',
+                      'outside_air_temperature',
+                      'AAT_dome_air_temperature',
+                      'AAT_mirror_temperature',
+                      'AAT_dome_dewpoint',
+                      'outside_humidity',
+                      'barometric_pressure',
+                      'wind_speed',
+                      'wind_gust',
+                      'wind_direction',
+                      'AAT_dome_door_status', # closed = 0 | partially open = 2 | open = 4
+                      'rain_sensor', # no rain = 0 | rain > 0
+                      'outside_dewpoint',
+                      'sky-ambient',
+                      'sky-ambient_uncertainty',
+                      'boltwood_sky_brightness', # dark = 0 | light > 0
+                      'boltwood_rain_flag', # no rain = 0 | rain > 0
+                      'boltwood_wet_flag', # dry = 0 | wet > 0
+                      'rain_since_9am',
+                      'SQM_sky_brightness']
+      # be sure to have the same number of entries in column_names and column_units
```

Comment:
If column names and units always have to match you can make them a dictionary instead of two lists:

```
columns:
    time_UTC: u.dimensionless_unscaled,
    outside_air_temparature: u.Celsius
    ...etc
```

You will then have to read it in slightly different, getting the `.keys()` or `.values()` (or `.items()`) depending on how you use it.
 
This would be preferred as it would enforce matching entries and make it a little more obvious what goes with what.